src/erlcloud_json.erl

```diff
@@ -0,0 +1,45 @@
+-module(erlcloud_json).
```

Comment:
the thing is that it uses json, and not xml here, so `erlcloud_xml` won't work for us